### DISCUSSION TITLE
Is it possible to distinguish between a New User and a Registered User

### DISCUSSION THREAD

**Discussion by G-Nation** [gladly-team/next-firebase-auth#515] [Q&A]
I followed up the example and i have an app with the StyledFirebaseAuth for the sign in and the sign up process.
Is it possible to distinguish between the two processes? I wanted to redirect the new users to a form were they give some extra info after the sign up process.

**Comment by kmjennison** [comment]
This is more a general Firebase question. Some approaches that come to mind:
* On sign-in, query your user database to see if they've completed the sign-up process and redirect based on it
* Set a custom claim on the user when they've completed sign-up and use it to authorize/redirect
